docker-compose.yml

```diff
@@ -21,6 +21,6 @@ version: '3'
 services:
   yunikornwebapp:
     build: .
-    image: yunikorn/yunikorn-web
+    image: apache/yunikorn
```

Comment:
could we use ` yunikorn/yunikorn:web-latest` here?

Reply:
you are right. I will change it